MekHQ/docs/history.txt

```diff
@@ -19,6 +19,7 @@ v0.45.2-SNAPSHOT
 + PR #882: Adds support for gzipped Campaign files (*.cpnx.gz).
 + Planets: Reunification War data for Outworlds Alliance + start of RWR 
 + PR #959: Fix for inability to repair damaged VTOL internal structure after loading campaign
++ Issue #967: Acquired Units can never be paid for
```

Comment:
We do not update history.txt as part of our PR process, one of the devs will update it after your PR lands.